components/reusable-components/reusable-table/FetchData.tsx

```diff
@@ -1,11 +1,18 @@
-const FetchData = async (): Promise<any> => {
-  const url = 'https://mocki.io/v1/ed331a85-c472-40e5-97eb-c15aedd6b8f2';
+export interface User {
+  id: string;
+  first_name: string;
+  last_name: string;
+  role: string;
```

Comment:
so here the role would be the enum you have defined in other PRs. Just have that in mind